src/components/forms/form-login/form-login.ts

```diff
@@ -16,21 +17,40 @@ export class loginFormAll extends Form {
           ".submit-message"
         ) as HTMLElement;
         if (!this.isValid) {
-          submitMessage.textContent = "Заполните все нужные поля";
-        } else if (!!this.isValid) {
+          const inputlist = document.querySelectorAll("input");
+          console.log(inputlist.length);
+          let truecounts = 0;
+          for(let i = 0; i < inputlist.length; i++) {
+            if(inputValidation(inputlist[i], true) == "true") {
+              truecounts += 1;
+            }
+          };
```

Comment:
```suggestion
              const truecounts = inputList
                         .map(input => inputValidation(input, true))
                         .filter(result => result === "true")
```

Думаю, что так выглядит проще

Reply:
Сделала вот так, с length в качестве считалки. (у меня валидатор отдает не true/false, а несколько состояний).